presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/base/MainActivity.java

```diff
@@ -9,20 +9,28 @@
 import com.globant.equattrocchio.data.ImagesServicesImpl;
 import com.globant.equattrocchio.domain.GetLatestImagesUseCase;
 import com.globant.equattrocchio.domain.GetSpecificImageUseCase;
+import com.globant.equattrocchio.domain.SaveImagesUseCase;
+
+import io.realm.Realm;
 
 public class MainActivity extends Activity {
 
     private ImagesPresenter presenter;
 
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        Realm.init(this);
```

Comment:
Specify a custom application, and move this line to this

Reply:
done